docs/generators/openapi-yaml.md

```diff
@@ -11,4 +11,4 @@ sidebar_label: openapi-yaml
 |ensureUniqueParams|Whether to ensure parameter names are unique in an operation (rename parameters that are not).| |true|
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
 |prependFormOrBodyParameters|Add form or body parameters to the beginning of the parameter list.| |false|
-|outputFile|output filename| |null|
+|outputFile|output filename| |openapi/openapi.yaml|
```

Comment:
this file is generated and shouldn't be modified directly

Reply:
Can you PLS tell me where is it generated and how?

Reply:
The thing is that outputFile default value in documentation is not correct. it should be `openapi/openapi.yaml`.

Reply:
i think it should be changed here:
https://github.com/OpenAPITools/openapi-generator/blob/9e391efd1d1f2c62ad9b8b9348d650f72062e3d3/modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/ConfigHelp.java#L55-L57

Reply:
default value: https://github.com/OpenAPITools/openapi-generator/blob/1bf8c15206c290e68051f65cc6fe1e1ed76ecb6d/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/OpenAPIYamlGenerator.java#L39

Reply:
and make sure to run `./bin/utils/export_docs_generators.sh` and commit the changed file

Reply:
I see, CLI option is not fully described in OpenAPIYamlGenerator.java. It is missing default value declaration.

Reply:
Here you are. Commit-ed openapi-yaml.md is the one created by `./bin/utils/export_docs_generators.sh`.

Reply:
great, you found the right file, I just noticed my suggestion which file to change was wrong ;-)